### PR TITLE
Number conversion

### LINKED ISSUE #5: 2-number conversion
Write a program to convert a number `N` from `base 10` to `base n`.

1. Generate a random number `n`.
2. Generate a random integer `N` in base 10.
3. Convert the integer `N` to `base n`, and output the result.

## Things to consider

1. Make sure the value of `n` is a positive integer between 2 and 16 (`n` should not be 10)
2. Make sure that `N` is a positive integer less than INFINITY

### PR DESCRIPTION
**Pull request by emmanueltukpe**
Resolved #5

### CODE REVIEW

File: words_in_reverse.py [hunk 1]
```diff
@@ -0,0 +1,8 @@
+ten_word_string = "This is the first code that we are gonna write"
+string_length = len(ten_word_string)
+sliced_string = ten_word_string[string_length::-1]
```
Comment (jirevwe) [inline words_in_reverse.py]: This file was merged yesterday, remove it and reopen another PR

**Review by jirevwe — CHANGES_REQUESTED** [review]

**Comment by jirevwe** [comment]
Remove the `words_in_reverse.py` and reopen another PR
It was merged from yesterday

**Comment by emmanueltukpe** [comment]
Removed the words_in_reversed.py

**Review by jirevwe — APPROVED** [review]

**Comment by jirevwe** [comment]
Create a new PR without that second file. I can't rebase the branch cleanly

**Comment by jirevwe** [comment]
Also, you can create a new branch, seems the issue is from the branch itself

**Pull request closed without merging**